news/2017-11-28-beatmap-spotlights-october-2017.md

```diff
@@ -0,0 +1,106 @@
+
+You don't usually see rhythm game songs mapped in this fashion, but it's exactly why [Nyquill](https://osu.ppy.sh/u/682935)'s rendition of Lilieze to Enryuu Laevateinn is so interesting. A style reminiscent of the old 2014 days, it still manages to hold up to this day. Despite not conforming to modern visual standards, the movements and feelings evoked from the map are still refined for the player, giving it a natural and smooth experience. Similarly, [Giralda](https://osu.ppy.sh/u/1823446)'s and [captin1](https://osu.ppy.sh/u/689997)'s guest difficulties are a fantastic complement to the set, with their styles being similar to one another. All in all, a deserving candidate for this month's Spotlights.
+
+[![](/wiki/shared/news/2017-11-28-beatmap-spotlights-october-2017/koi-kou-enishi.jpg)](http://osu.ppy.sh/s/622780)
+
+[Crystal](https://osu.ppy.sh/u/1646397) brings us yet another top tier mapset this month for KOTOKO's Koi Kou Enishi. A wide selection of Insane and Expert difficulties are available here, but don't let the star ratings confuse you - while the highest difficulty by [yf_bmp](https://osu.ppy.sh/u/1243669) may be below five stars, it features quite a few spaced streams and patterns that you wouldn't find on the average map!
+
+[![](/wiki/shared/news/2017-11-28-beatmap-spotlights-october-2017/punk-rock-girl.jpg)](https://osu.ppy.sh/s/577394)
+
+[Milan](https://osu.ppy.sh/u/1152994)'s latest mapset of Punk Rock Girl brings us an exhilarating experience suitable for all ranks! You won't be seeing any jumps here, but rather a constant barrage of sliders and circles all mapped in an orthodox alternating fashion, all intended to keep you on your toes in this short song. Guest Difficulties from both [Pho](https://osu.ppy.sh/u/3624692) and Streetlight Manifesto enthusiast  [pishifat](https://osu.ppy.sh/u/3178418) bring extra zest into the set, with interesting rhythms and patterns not usually seen in their respective difficulties.
+
+## osu!taiko
+
+[![](/wiki/shared/news/2017-11-28-beatmap-spotlights-october-2017/zadamaga.jpg)](https://osu.ppy.sh/s/458476)
+
+Leading this month's osu!taiko Spotlights we have ZADAMGA, a one-time masterpiece that you would not see normally in the depths of the Ranked Beatmaps category. Consisting of a total of six difficulties mapped by [agu](https://osu.ppy.sh/u/1119122) himself, not only does it demonstrate an exemplary skill in spread management in lower difficulties, but also an outstanding mastery in different rhythm and beat snap usage by the mapper.
+
+While simple 1/4 and 1/6 patterns are most common in modern osu!taiko mapping, the highest difficulty “Fatale Oni” employs a beautifully mixed repertoire of simple and compound rhythms, having an almost equalized usage of 1/3 rhythms in comparison to regular 1/2 or 1/4 rhythms. In short, a very complete mapset that has its well deserved spot on the top of this month's Spotlights!
```

Comment:
something similar to above
this pr -> `“` and `”`
but my keyboard -> `"`

🤷‍♂️

Reply:
Again, use `"` for now.